LSIAgent.m

```diff
@@ -0,0 +1,30 @@
+//
+//  LSIAgent.m
+//  NOCList-Objc
+//
+//  Created by Kenny on 6/8/20.
+//  Copyright © 2020 Hazy Studios. All rights reserved.
+//
+
+#import "LSIAgent.h"
+
+@implementation LSIAgent
+
+- (instancetype)initWithCoverName:(NSString *)coverName realName:(NSString *)realName accessLevel:(int)accessLevel copromised:(BOOL)compromised
+{
+    self = [super init];
+    if (self) {
+        _coverName = coverName;
+        _realName = realName;
+
+        _accessLevel = [NSNumber numberWithInt: accessLevel];
```

Comment:
I prefer not to use a space after `:`, but I put a space after variable name `accessLevel` if there are multiple parameters.

```objc
_accessLevel = [NSNumber numberWithInt:accessLevel];
```